packages/forklift-console-plugin/src/modules/Plans/views/details/components/SettingsSection/components/NameTemplate/NameTemplateInput/NameTemplateInput.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { Dispatch, FC, SetStateAction } from 'react';
+import { ModalInputComponentType } from 'src/modules/Providers/modals';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import { Radio, TextInput } from '@patternfly/react-core';
+
+import { NameTemplateRadioOptions } from '../utils/constants';
+
+type InputRendererProps = {
+  value: string;
+  onChange: (string: string) => void;
+};
+
+const NameTemplateInputFactory: (
+  selected: NameTemplateRadioOptions,
+  setSelected: Dispatch<SetStateAction<NameTemplateRadioOptions>>,
+) => ModalInputComponentType = (selected, setSelected) => {
+  const { t } = useForkliftTranslation();
+
+  const InputRenderer: FC<InputRendererProps> = ({ value, onChange }) => (
+    <>
+      <Radio
+        isChecked={selected === NameTemplateRadioOptions.defaultNameTemplate}
+        name="name-template"
+        onChange={() => setSelected(NameTemplateRadioOptions.defaultNameTemplate)}
+        label={t('Use default naming template')}
+        id="default-naming-template"
+      />
+      <Radio
+        isChecked={selected === NameTemplateRadioOptions.customNameTemplate}
+        name="name-template"
```

Comment:
same name?

Reply:
for PF radio to work as a radio we need same name

Reply:
what? something doesn't make much sense... its different elements, name is property of identifications, not a type... please double check

Reply:
Yes I'm sure, adding screenshots: (different name makes both radio selectable since they don't share the same name)
![radio-diff-name](https://github.com/user-attachments/assets/6fe79680-946c-467e-84ab-858b2529d16b)
![radio-same-name](https://github.com/user-attachments/assets/38a7f258-2be9-4d53-a33a-f26e0b141f99)

please follow props docs: https://v5-archive.patternfly.org/components/forms/radio#radio 
clearly they say name is for the name of group of radios